### PR TITLE
Create aks.cdoc

### PR DESCRIPTION
**Pull request by ajbisht**
**Release Note**
```release-note
NONE
```

### CODE REVIEW

**Review by oscerd — APPROVED** [review]
Thanks. It needs some rework but it's a good start

File: docs/modules/ROOT/pages/installation/platform/aks.cdoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
+Apache Camel K is a lightweight integration platform, born on Kubernetes,
```
Comment (tadayosi) [inline docs/modules/ROOT/pages/installation/platform/aks.cdoc:1]: No other install guides start from plain sentence. We should start from title.

File: docs/modules/ROOT/pages/installation/platform/aks.cdoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+with serverless superpowers.
+
+# Camel K deployment in AKS
```
Comment (tadayosi) [inline docs/modules/ROOT/pages/installation/platform/aks.cdoc:4]: It should be more reader-friendly to tell what AKS stands for like other guides.

File: docs/modules/ROOT/pages/installation/platform/aks.cdoc [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+# Camel K deployment in AKS 
+
+## Prerequisites
+
+- AKS cluster with cluster role binding to deploy operator
+- Container Image Registry installed and configured for pull (using nexus)
+- Base image to build camel integration push to Image Registry (jdk-jre image present in nexus)
+
+
+Installation using Kamel CLI
```
Comment (tadayosi) [inline docs/modules/ROOT/pages/installation/platform/aks.cdoc:13]: Maybe you omitted the beginning `##`?

File: docs/modules/ROOT/pages/installation/platform/aks.cdoc (hunk 1 above)
Comment (tadayosi) [inline docs/modules/ROOT/pages/installation/platform/aks.cdoc:1]: Looks good, but to begin with the file name should be `aks.adoc` not `.cdoc`.

**Review by tadayosi — CHANGES_REQUESTED** [review]

**Comment by oscerd** [comment]
I think we should merge this and rework. Otherwise it will be lost.

**Comment by tadayosi** [comment]
I don't mind it if the author is inactive.

**Comment by ajbisht** [comment]
Waiting for PR#2845 to be merged

**Comment by oscerd** [comment]
The PR #2845 is not aligned and it's not updated from a while, not sure it will be merged soon. @astefanutti can we switch to go 1.17?

**Comment by astefanutti** [comment]
> The PR #2845 is not aligned and it's not updated from a while, not sure it will be merged soon. @astefanutti can we switch to go 1.17?

Generally, the Go version is driven by the version that is used by the Kubernetes dependencies we have. It could be possible to upgrade to 1.17, but if there is no special requirement, it's generally safer to follow Kubernetes.

That being said, I fail to see the relationship between this PR and #2845.

**Comment by oscerd** [comment]
> > The PR #2845 is not aligned and it's not updated from a while, not sure it will be merged soon. @astefanutti can we switch to go 1.17?
> 
> Generally, the Go version is driven by the version that is used by the Kubernetes dependencies we have. It could be possible to upgrade to 1.17, but if there is no special requirement, it's generally safer to follow Kubernetes.
> 
> That being said, I fail to see the relationship between this PR and #2845.

The original issue was related to some CVE in reported in Azure while deploying camel-k, updating to go 1.17 seems to be the fix for them. So, I asked the reporter to write a guide on how to deploy on AKS, he opened this PR, and then commented here about waiting for the Go upgrading.

**Comment by astefanutti** [comment]
> > > The PR #2845 is not aligned and it's not updated from a while, not sure it will be merged soon. @astefanutti can we switch to go 1.17?
> > 
> > 
> > Generally, the Go version is driven by the version that is used by the Kubernetes dependencies we have. It could be possible to upgrade to 1.17, but if there is no special requirement, it's generally safer to follow Kubernetes.
> > That being said, I fail to see the relationship between this PR and #2845.
> 
> The original issue was related to some CVE in reported in Azure while deploying camel-k, updating to go 1.17 seems to be the fix for them. So, I asked the reporter to write a guide on how to deploy on AKS, he opened this PR, and then commented here about waiting for the Go upgrading.

I see, this relates to #3113. If there are no incompatibility, we can upgrade Go even if the Kubernetes version that's currently use still uses 1.16. I can see it could possibly solve CVE-2022-24407, but for CVE-2022-24407 it's difficult to correlate the fix to a Go upgrade.

**Review by claudio4j — CHANGES_REQUESTED** [review]

**Comment by oscerd** [comment]
If this is not active anymore, we shoud close, align or merge.

**Pull request closed without merging**